### ISSUE TITLE
Dealer vanished then save broke

### ISSUE THREAD

**Issue by Bear8080** [igorkis-scrts/A3-Antistasi-Plus#30]
**Describe the bug**
When playing on Tanoa, we noticed pretty early on that the dealer was gone after we had already found him with intel. We went several times to check after that and he was not there. We tried to use the "Restart dealer" thing in the commander menus but literally nothing happened. After that, we had to stop to restart the server for windows updates and when we tried to load, it was completely broken.

**To Reproduce**
Steps to reproduce the bug (If you can't reproduce it, just give details of what happened beforehand. Anything you think could be relevant):
No clue at all, we were just playing normally.

**Expected behaviour**
The dealer to not disappear and the save to not brick.

**Screenshots**

**Environment (please complete the following information):**
Version: 1.5.2
Map (specify if Altis Blufor): Tanoa

* [ ] LAN Hosted
* [ x] Private dedicated server
* [ ] Private dedicated server with Headless client(s)
* [ ] Community Server 1
* [ ] Community Server 2
* [ ] Community Test Server
* [ ] Community Test Server 2

Modset: 
[VALCO.html.txt](https://github.com/igorkis-scrts/A3-Antistasi-Plus/files/6501361/VALCO.html.txt)

Time the bug occurred (Please leave in GMT+0): ~12:00PM GMT
If **Not** on a community server, please leave the time zone of your client for LAN, or the server for dedicated: 

**Additional context**
Windows Server 2019 64bit OS
Intel Xeon E3-1275 v6 CPU
64GB 2400MHz DDR4 ECC Buffered RAM
1TB SSD

If you were LAN hosting, please attach your client .RPT file from when the bug occurred, If on a private dedicated server, please attach the server's .RPT from when the bug occurred.
[arma3server_x64_2021-05-18_12-32-14.rpt.txt](https://github.com/igorkis-scrts/A3-Antistasi-Plus/files/6501332/arma3server_x64_2021-05-18_12-32-14.rpt.txt)

**Comment by igorkis-scrts** [comment]
Tanoa? Antistasi Plus does not support Tanoa at this moment, only officially released missions are supported. 

You clearly doesn't added all Antistasi Plus stuff on your Tanoa port, so the game brokes itself from very beginning.
```
2021/05/18, 12:37:15 Error in expression <owDamage false;
mapX allowDamage false;
traderScreenX allowDamage false;
teamPla>
2021/05/18, 12:37:15   Error position: <traderScreenX allowDamage false;
teamPla>
2021/05/18, 12:37:15   Error Undefined variable in expression: traderscreenx
```

`traderScreenX` exist on all Antistasi Plus-supported maps.

**Comment by igorkis-scrts** [comment]
About dealer disappearance bug - it is different bug that is already fixed in the upcoming Antistasi Plus 1.5.3.

**Comment by Bear8080** [comment]
It was working fine for a couple of saves and restarts, trader was there and it was working but seemingly randomly vanished at some point. I now understand it is not officially supported but I put everything in properly from the beginning and it was actually working flawlessly until that point.

**Comment by igorkis-scrts** [comment]
Arms Dealer disappearance fixed in 1.5.3, Tanoa is not supported yet.
